src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person/Person';
```

Comment:
Ensure that the 'Person' component is correctly imported from './components/Person/Person'. Verify that the path is correct and that the component is properly exported from that location.